docs/source/command-line-scripts.md

```diff
@@ -0,0 +1,316 @@
+                                            [--base_variables BASE_VARIABLES] 
+                                            [--time_dim TIME_DIM] 
+                                            [--time_start TIME_START] 
+                                            [--time_stop TIME_STOP] 
+                                            [--levels LEVELS] 
+                                            [--averaging_dims AVERAGING_DIMS]
+                                            [--runner RUNNER] 
+```
+
+_Command options_:
+
+* `--input_path`: (required) Input Zarr path
+* `--output_path`: (required) Output Zarr path
+* `--base_variables`: Comma delimited list of variables in --input_path. Each variable VAR results in a VAR_zonal_power_spectrum entry in --output_path. Default: `geopotential,specific_humidity,temperature`
+* `--time_dim`: Name for the time dimension to slice data on. Default: `time`
+* `--time_start`: ISO 8601 timestamp (inclusive) at which to start evaluation. Default: `2020-01-01'`
+* `--time_stop`: ISO 8601 timestamp (inclusive) at which to stop evaluation. Default: `2020-12-31`
+* `--levels`: Comma delimited list of pressure levels to compute spectra on. If empty, compute on all levels of --input_path. Default: `500,700,850`
+* `--averaging_dims`: Comma delimited list of variables to average over. If empty, do not average. Default: `time`
+* `--runner`: Beam runner. Use `DirectRunner` for local execution.
+
+*Example*
+
+```bash
+python wb2_compute_zonal_power_spectrum.py \
+  --input_path=gs://weatherbench2/datasets/era5/1959-2022-6h-240x121_equiangular_with_poles_conservative.zarr  \
+  --output_path=PATH \
+  --time_start=2020 \
+  --time_stop=2020 \
+  --base_variables=geopotential,specific_humidity,temperature,u_component_of_wind,v_component_of_wind,wind_speed,10m_u_component_of_wind,10m_v_component_of_wind,10m_wind_speed,2m_temperature,mean_sea_level_pressure,total_precipitation_6hr,total_precipitation_24hr
+```
+
+## Compute ensemble mean
+
+To use the ensemble mean in deterministic evaluation, we first must compute the mean and save it in a separate file.
+
+```
+usage: wb2_compute_ensemble_mean.py [-h] 
+                                    [--input_path INPUT_PATH] 
+                                    [--output_path OUTPUT_PATH]
+                                    [--realization_name REALIZATION_NAME]
+                                    [--runner RUNNER] 
+```
+
+_Command options_:
+
+* `--input_path`: (required) Input Zarr path
+* `--output_path`: (required) Output Zarr path
+* `--realization_name`: Name of realization/member/number dimension. Default: `realization`
+* `--runner`: Beam runner. Use `DirectRunner` for local execution.
+
+*Example*
+
+```bash
+python ensemble_mean.py -- \
+  --input_path=gs://weatherbench2/datasets/ens/2018-64x32_equiangular_with_poles_conservative.zarr \
+  --output_path=PATH \
+  --realization_name=number
+```
+
+## What about regridding?
```

Comment:
We have the regridding script checked in now, so we could mention it

Reply:
Added.